src/DisplayButtons.js

```diff
@@ -0,0 +1,7 @@
+import {CustomButtons} from './CustomButtons'
+
+export function DisplayButtons(){
+   return <div className='flex justify-center space-x-5'>
+        {Array(3).fill(null).map((v,i)=> <CustomButtons btnID={i+1}/>)}
```

Comment:
Use better parameter names; Knowing what `v` represents is hard. Also, add keys on the mapped items.